drivers/volume/portworx/portworx.go

```diff
@@ -1453,6 +1454,77 @@ func (d *portworx) ValidatePureFaFbMountOptions(volumeName string, mountoption [
 
 }
 
+func (d *portworx) ValidatePureFaCreateOptions(volumeName string, FStype string, volumeNode *node.Node) error {
+	// Checking if file systems are properly set
+	FScmd := fmt.Sprintf(mountGrepVolume, volumeName)
+	FSout, err := d.nodeDriver.RunCommandWithNoRetry(
+		*volumeNode,
+		FScmd,
+		node.ConnectionOpts{
+			Timeout:         crashDriverTimeout,
+			TimeBeforeRetry: defaultRetryInterval,
+		})
+	if err != nil {
```

Comment:
please add err to the log

Reply:
Added err to all returning error for this function.